consensus/common/src/main/java/org/hyperledger/besu/consensus/common/PoAMetricServiceImpl.java

```diff
@@ -25,7 +25,7 @@
 public class PoAMetricServiceImpl implements PoAMetricsService {
 
   private final BlockInterface blockInterface;
-  private final Blockchain blockchain;
+  protected final Blockchain blockchain;
```

Comment:
why `protected`?

Reply:
We can safely introduce protected (and possibly final) accessors for `Blockchain` in `PoAMetricServiceImpl` which can be used by subclasses.

Reply:
done